pages/learn/develop/local-mode.md

```diff
@@ -1,146 +1,224 @@
+Local mode is the development mode for {{ $names.company.lower }}. It allows you to build and sync code to a single development device in your local network without having to go through the {{ $names.cloud.lower }} build service and deployment pipeline. It uses the Docker daemon on the device to build container images, and then the device Supervisor starts the containers in the same way as if it were deployed via the cloud.
 
 ## Local mode requirements
 
-In order to use local mode on a device:
-- The device must be running balenaOS v2.29.0 or higher.
+To use local mode on a device:
+
+- The device must be running {{ $names.os.lower }} v2.29.0 or higher.
 - The device must be running a [development][development] variant of the OS.
-- You must have the [balena CLI][cli] installed on your workstation.
-- Local mode must be enabled through the dashboard. To use local mode on a development device, click on the small *Actions* dropdown at the top right of the device page and select *Enable Local Mode*.
+- You must have the [{{ $names.company.lower }} CLI][cli] installed on your development machine.
+- Local mode must be enabled through the {{ $names.cloud.lower }} dashboard. You can enable it from either the *Actions* menu of the device dashboard or click to expand the arrow located on the device dashboard and select *Enable Local Mode*.
+
+![Enable local mode](/img/local-mode/enable-local-mode.png)
+
+## Local mode caveats
+
+- In local mode, a device will not send logs back to the {{ $names.cloud.lower }} dashboard. Refer to the [local mode logs section](#local-mode-logs) to view logs in local mode.
+- Device and service environment variables set from the {{ $names.cloud.lower }} will not be applied to local mode containers. It is still possible to set environment variables in your `docker-compose.yml` or `Dockerfile`.
+- Changes to device [configuration variables][configuration], for example, `RESIN_HOST_CONFIG_gpu_mem`, will result in the device rebooting and applying those settings.
+- Actions such as _Restart_ and _Purge data_ from the {{ $names.cloud.lower }} dashboard will not apply to local mode containers.
+- When switching out of local mode and back to tracking releases from {{ $names.cloud.lower }}, the Supervisor will destroy any local mode containers and  volumes, as well as clean up unneeded base images, and then start up the application that {{ $names.cloud.lower }} instructs it to run.
 
-<img src="/img/local-mode/device-in-local-mode.png" width="100%">
+![Device in local mode](/img/local-mode/device-in-local-mode-20-01-09.png)
 
 ## Scan the network and find your device
```

Comment:
It's probably worth noting that this information can be found on the dashboard, with the ip address generally being correct for how you would access it. The <short uuid>.local should always work without a scan as well. This decreases the time for getting started